You also have some of your own YAML coding norm issues here:

```
./.github/workflows/discover-tier1-hofx.yml
  12:1      error    trailing spaces  (trailing-spaces)

./.github/workflows/discover_nightly.yml
  7:1       error    trailing spaces  (trailing-spaces)
```

--- a/.github/workflows/discover-tier1-hofx.yml
+++ b/.github/workflows/discover-tier1-hofx.yml
@@ -9,6 +9,6 @@ defaults:
     shell: bash
 
 jobs:
-  
+
   swell-discover-tier1-hofx:
     uses: GEOS-ESM/CI-workflows/.github/workflows/swell-discover-tier1-hofx.yml@main

--- a/.github/workflows/discover_nightly.yml
+++ b/.github/workflows/discover_nightly.yml
@@ -4,7 +4,7 @@ name: discover-nightly
 
 on:
   workflow_dispatch:
-  
+
   schedule:
     # * is a special character in YAML so you have to quote this string
     - cron:  '0 0 * * *'

